### ISSUE TITLE
HydrogeoUnit

### ISSUE THREAD

**Issue by mbeaufils** [opengeospatial/Geotech#13]
Source definition : OGC GroundWaterML2, also candidate for IFC

A large hydrogeologically defined body of ground typically consisting of hydraulically connected hydrogeological units, whose waters are flowing to a common or multiple outlets, and which is delimited by a groundwater divide.

**Comment by IHalfon** [comment]
I don't see the use in geotechnics ? example ?

**Comment by jjkaelin** [comment]
HydrogeoUnit: the use of Aquifer and GroundwaterBasin should be considered, being more understandable.

**Comment by dponti** [comment]
Yes it does seem that hydrogeounit may not be necessary when we have two common terms to define the types of hydrogeological units included in the definition (aquifer and groundwater basin). Perhaps we should instead consider if there's value in defining a "hydrogeologic system" rather than a unit, that incorporates both the groundwater basin and the aquifer systems it contains. Boundaries of a groundwater basin can be modified by pumping, whereas the hydrologic properties of aquifers are generally static (unless altered by overpumping induced compaction) so consideration of the system might be warranted.

**Comment by neilchadwick-dg** [comment]
I'm not an expert in hydrogeology, but I've worked with hydrogeologists so have some experience. Based on this and a little bit of research here are my thoughts...

I quite like this definition - perhaps oversimplified but easy to understand...

*A hydrogeological unit is defined as any soil or rock unit or zone that by virtue of its hydraulic properties has a distinct influence on the storage or movement of groundwater.*  ([source](https://open.canada.ca/data/en/dataset/34c0dbf2-9595-84f3-679c-7d2d7c90ecfel))

Geological units normally form the basis of hydrogeological units, but like geotechnical units the mapping is not always 1:1. A hydrogeological unit may combine geological units, or a geological unit (e.g. with different lithologies) may be split into different hydrogeological units.

Hydrogeological units are a critical part of the hydrogeological model.

Hydrogeological units are NOT bodies of water or groundwater, as such. These exist separately, albeit influenced by the hydrogeological units of course. I'll comment on those under the relevant headings.

The only thing that bothers me is that at the moment we are not differentiating between hydrogeological units in an observational model and hydrogeological units in an analytical (design - synthesis) model, whereas we have distinct geological (observational model) and geotechnical units (design model). Slightly inconsistent. Not sure what the answer is, or whether it matters.
